---
title: "Parcellation of the mouse brain using molecular imaging"
author: Alex Lee
bibliography: references.bib
csl: nature.csl
format:
  pdf:
    toc: false
    number-sections: true
    link-citations: true
    colorlinks: true
    geometry:
    - top=19.05mm
    - left=19.05mm
    - right=19.05mm
    - bottom=31.75mm
---

# Specific Aims

&nbsp;&nbsp;&nbsp;&nbsp;Parallel technological advances in molecular biology and microscopy have dramatically increased the ease with which experimental biologists can make spatially resolved measurement of large numbers of proteomic or transcriptomic features. These high-fidelity molecular mapping techniques are promising to catalog the enormous diversity of cell types in the complex organs like the brain, which has been recognized since the early work of Ramon y Cajal [@y_cajal_texture_1999]. These new capabilities are the focus of large scale collaborations facilitated by the NIH BRAIN initiative[@noauthor_nih_2022], which aim to precisely elucidate the structural oragnization of the brain by generating multimodal, cross-species datasets for comparison. The datasets that will result from these projects will be unprecedented in size and complexity. They will require new and innovative computational techniques to organize and analyze, but will transform our understanding of the molecular and cellular organizing principles of the brain.

&nbsp;&nbsp;&nbsp;&nbsp;Two principal focus areas in the neuroscience community's investigation of these relationships are variation in gene expression[@tasic_shared_2018] and connectivity[@oh_mesoscale_2014; @sun_integrating_2021], and how they interact in the circuitry of the brain. A variety of unsupervised and supervised machine learning techniques have been developed for extraction of latent features for clustering or subregion detection in spatial imaging data[@walker_deciphering_2022], but these methods typically target 2D tissue sections, making them unsuited for analysis of 3D volumetric data. In addition, there are few techniques suited for multimodal data integration in the specific case of connectivity and transcriptomic data, with most methods focus on spatial transcriptomics exclusively. Although single-cell RNA sequencing (scRNA-seq) has been enormously impactful for our ability to catalog neuronal variation, it is unclear how subclasses observed in scRNA-seq correspond to graded connectivity differences, such as those found in CA1 of the hippocampus[@cembrowski_dissociable_2018].

As significant heterogeneity has been observed in transcriptomic measurements of neuronal cells, 

&nbsp;&nbsp;&nbsp;&nbsp;I propose to address this unmet need by developing computational methods for machine learning-based spatial subregion detection, with application to existing and in-development datasets from the Allen Institute for Brain Science and the Broad Institute. I will use these techniques to map the organization of functional subregions in spatial transcriptomics datasets in the mouse brain, and use interpretable statistical techniques to understand which cell type contribute most to these subregions. I will then make use of these techniques to integrate this spatial transcriptomics dataset with the existing Allen Mouse Connectivity Atlas[@oh_mesoscale_2014], identifying genetic correlates of differential connectivity in transcriptomically similar cells.

## Aim 1: Establish a spatial transcriptomics data analysis pipeline to identify and characterize novel subregions in the mouse brain

&nbsp;&nbsp;&nbsp;&nbsp;First, I will develop a novel interpretable machine learning pipeline for unsupervised and interpretable discovery of functional subregions in large scale, 3D spatial transcriptomics data[@langlieb_cell_2023; @yao_high-resolution_2023]. Next, I will characterize whether these subregions are driven by specific cell-types or gene expression gradients across cells using regression analysis. Determination of the principal gene expression organization spatial patterns in these datasets will pave the way towards understanding the circuitry of the mouse brain.

## Aim 2: Determine transcriptomic and non-neuronal cell-type determinants of differential connectivity across the brain by integrating the Allen Mouse Connectivity Atlas with spatial transcriptomics datasets

&nbsp;&nbsp;&nbsp;&nbsp;First, I will a develop predictive regression model for association of connectivity patterns in the Allen Mouse Connectivity Atlas[@oh_mesoscale_2014] with cell-type counts and transcriptomic features identified in Yao et al.[@yao_high-resolution_2023] and Langlieb et al[@langlieb_cell_2023]. I will use this model to identify whether there are significant genes or gene modules that are predictive of differential connectivity among neurons. I will also investigate whether specific non-neuronal populations are predictive of differential connectivity of the same neuronal populations. These analyses will reveal the impact of cell-type and transcriptomic variation on the structural connectivity of the mouse brain. 

## Significance: 
 
something very important 

# Background

&nbsp;&nbsp;&nbsp;&nbsp;From the first neuroanatomical studies[@y_cajal_texture_1999], it has been recognized that although cells can be broadly grouped into high-level categories (like neurons, or glia), even visual inspection of silver-stained cells is sufficient to identify a diverse array of cellular subtypes that differ in their morphologic and connectomic characteristics. Understanding the role of this diversity in facilitating the complex computations the brain performs remains one of the fundamental projects of modern neuroscience research. However, the complexity of this project has increased in the modern 'omics age due to the accessibility of high dimensional molecular measurement tools. In particular, the ubiquity of high-fidelity molecular imaging data provides a unique opportunity to study the correspondence between our current and previous understanding of the organizing principles of brain structure and the patterns that can be identified in new datasets using statistical machine learning techniques. 

&nbsp;&nbsp;&nbsp;&nbsp;Large scale molecular atlases such as the Allen Brain Atlas[@lein_genome-wide_2007] (ABA) and Allen Mouse Connectivity Atlas[@oh_mesoscale_2014] (ACA) have been critical tools to study the molecular organization of the brain. The integration of automated high-throughput data collection methods, comprehensive informatics, and fully open data sharing have facilitated an enormous number of recent studies (as measured by the 3,474 citations of the original Allen Brain Atlas Paper [as of 2023-02-18]). These datasets were integral to the development of the Allen Common Coordinate Framework (CCF) v3[@wang_allen_2020], a systematic effort to integrate gene expression, connectivity, transgenic expression, and histology in the form of a neuroanatomical consensus structural labeling. New datasets are rapidly emerging, such as from Yao et al.[@yao_high-resolution_2023], using MERFISH[@chen_spatially_2015], and Langlieb et al.[@langlieb_cell_2023], using Slide-SeqV2[@rodriques_slide-seq_2019], which offer spatial single-cell resolution. As the interpretation and mapping of these datasets into anatomical and functional knowledge (as in the CCF) has largely been driven by expert human knowledge, new and systematic computational techniques for multimodal atlas construction will be an important tool to provide neuroanatomical insight. 

&nbsp;&nbsp;&nbsp;&nbsp;Multimodal data integration is a key challenge for such computational methods, and is crucial to understanding neural structural organization principles. At the mesoscale, pioneering work from Chen et al.[@chen_modular_2022] has found that brain regions that have similar cell-type organization are likely to be connected. In a small number (~100) of single cells, an investigation by Economo et al. has displayed a high correspondence between single-cell transcriptomic clusters and projection patterning in layer 5 pyramidal tract neurons. However, particularly with the release of one-hemisphere or whole-brain mouse spatial transcriptomic data, researchers can now systematically investigate the impact of gene expression patterns and connectivity patterns as well as the correlation between non-neuronal cell density and connectivity patterns of single-cells. 

* Existing techniques for multimodal data integration / subregion discovery focus really on 2D, not suited for brain imaging data
* hypothesis: a small number of gene expression gradients form the hierarchical structure of the mouse brain and can be robustly identified from ST data; these expression gradients also template connectivity patterns across the brain

aim1: create new comp method for data driven atlas creation to identify key gradients in gene expression across brain 
* prelim work on non spatial transcriptomic test data

aim 1.1 computational method for v large, sparse ST data
aim 1.2 analysis of cell type contribution to these gradients

aim 2: identify correlates of differential connectivity in gene expression

aim 2.1 comp method for multimodal region discovery using connectivity and gene expr data
aim 2.2 regression analysis of transcriptomic/cell type contributors to differential connectivity







but existing techniques for spatial subregion detection are poorly suited for analysis of large-scale, 3D imaging data. For example, one common class of methods utilizes spatial Gaussian processes[@townes_nonnegative_2023] with $O(n^3)$ scaling, which is undesirable for large spatial (3D) imaging datasets. Other classes of promising methods would require assumptions to be made about the size of the spatial neighborhood for cellular interaction, which is difficult to assess with serial sections[@brbic_annotation_2022].



Single-cell sequencing studies[@tasic_shared_2018; @zeisel_molecular_2018; @bandler_single-cell_2022] in the mouse have made significant progress towards charting cell type gradients as defined by transcriptomic content; likewise, computational efforts have been developed for very fine-grained connectivity mapping and clustering[@knox_high-resolution_2018]. Molecular atlases such as the Allen Institute's Common Coordinate Framework[@wang_allen_2020] (CCF) atlas are critical to help organize inquiries into these phenomena by facilitating comparisons across studies and spatial scales. One strength of these atlases is their extensive manual curation, starting from the level of image registration and quality-checking towards consensus decisions by expert anatomists to delineate subregions and their boundaries.

# Aim 1: establish a method for the parcellation and analysis of transcriptomic imaging data

# Aim 2: establish an analysis pipeline for the integration of transcriptomic and connectomic imaging data 
