---
title: "Parcellation of the mouse brain using molecular imaging"
author: Alex Lee
bibliography: references.bib
csl: nature.csl
format:
  pdf:
    toc: false
    number-sections: true
    link-citations: true
    colorlinks: true
---

# Specific Aims

&nbsp;&nbsp;&nbsp;&nbsp;Parallel technological advances in molecular biology and microscopy have dramatically increased the ease with which experimental biologists can make spatially resolved measurement of large numbers of proteomic or transcriptomic features. These high-fidelity molecular mapping techniques are promising to catalog the enormous diversity of cell types in the complex organs like the brain, which has been recognized since the early work of Ramon y Cajal [@y_cajal_texture_1999]. These new capabilities are the focus of large scale collaborations facilitated by the NIH BRAIN initiative[@noauthor_nih_2022], which aim to precisely elucidate the structural oragnization of the brain by generating multimodal, cross-species datasets for comparison. The datasets that will result from these projects will be unprecedented in size and complexity and will require new and innovative computational techniques to organize and analyze. 

&nbsp;&nbsp;&nbsp;&nbsp;Two principal focus areas in the neuroscience community's investigation of these structure-function relationships are variation in gene expression[@tasic_shared_2018] and connectivity[@oh_mesoscale_2014; @sun_integrating_2021], and how they interact in the circuitry of the brain. A variety of unsupervised and supervised machine learning techniques have been developed for extraction of subregions from spatial imaging data[@walker_deciphering_2022], however these methods typically target 2D tissue sections, making them unsuited for analysis of 3D volumetric data. In addition, there are few techniques suited for multimodal data integration in the specific case of connectivity and transcriptomic data, with most methods focus on spatial transcriptomics exclusively. 

&nbsp;&nbsp;&nbsp;&nbsp;I propose to address this unmet need by developing computational methods for machine learning-based spatial subregion detection, with application to existing and in-development datasets from the Allen Institute for Brain Science. I will use these techniques to map the organization of functional subregions in spatial transcriptomics datasets in the mouse brain, and use interpretable statistical techniques to understand which cell type contribute most to these subregions. I will then make use of these techniques to integrate this spatial transcriptomics dataset with the existing Allen Mouse Connectivity Atlas[@oh_mesoscale_2014], identifying genetic correlates of differential connectivity in transcriptomically similar cells.

## Aim 1: establish a method for the parcellation and analysis of transcriptomic imaging data

&nbsp;&nbsp;&nbsp;&nbsp;I hypothesize that the mouse brain's subregions vary in terms of only a small number of the different cell types. To investigate this I will build on previous models for interpretable, stability-driven automatic subregion detection[@wu_stability-driven_2016] and extend them to the setting of spatial transcriptomics datasets, which are both high dimensional and very sparse. I will then characterize these subregions by analyzing the relative contribution of different cell types to these different identified regions.

## Aim 2: establish an analysis pipeline for the integration of transcriptomic and connectomic imaging data 

&nbsp;&nbsp;&nbsp;&nbsp;I will study the relationship between gene expression and connectivity by developing statistical methods to test whether particular genes are indicative of differential connectivity. I will benchmark methods on their ability to capture previously identified connectivity-genetic relationships and test whether single cell features or regional features are more predictive of differential connectivity. 

## Significance: 
 
some other stuff

# Background

&nbsp;&nbsp;&nbsp;&nbsp;From the first neuroanatomical studies[@y_cajal_texture_1999], it has been recognized that although cells can be broadly grouped into high-level categories (like neurons, or glia), even visual inspection of silver-stained cells is sufficient to identify a diverse array of cellular subtypes that differ along morphologic and connectomic characteristics. Understanding the role of this diversity in facilitating the complex computations the brain performs remains one of the fundamental projects of modern neuroscience research. However, the scope of this project has fundamentally changed in the modern 'omics age due to the accessibility of high dimensional molecular measurement tools. In particular, the ubiquity of high-fidelity molecular imaging data provides a unique opportunity to study the correspondence between our current and previous understanding of the organizing principles of brain structure and the patterns that can be identified in new datasets using modern statistical machine learning techniques. 

&nbsp;&nbsp;&nbsp;&nbsp;

&nbsp;&nbsp;&nbsp;&nbsp;The Allen Brain Atlas[@lein_genome-wide_2007] (ABA) and Allen Mouse Connectivity Atlas[@oh_mesoscale_2014] (ACA) are ideal datasets to computationally study this correspondence. The integration of automated high-throughput data collection methods, comprehensive informatics, and fully open data sharing have facilitated an enormous number of recent studies (as measured by the 3474 citations of the original Allen Brain Atlas Paper [as of 2023-02-18]), and in themselves were useful to characterize cellular heterogeneity not observed in classical histological neuroanatomical studies. These datasets were integral to the development of the Allen Common Coordinate Framework v3[@wang_allen_2020], a systematic effort to integrate gene expression, connectivity, and transgenic expression in the form of a generalized reference coordinate framework with neuroanatomical consensus structural labeling. 


&nbsp;&nbsp;&nbsp;&nbsp;These datasets will be augmented by new datasets using spatially resolved transcriptomic techniques, such as from Yao et al.[@yao_high-resolution_2023], using MERFISH[@chen_spatially_2015], and Langlieb et al.[@langlieb_cell_2023], using Slide-Seq[@rodriques_slide-seq_2019]. The single-cell resolution of this spatially-resolved transcriptomic data can now be used a reference for single-cell mapping, with potential to integrate a wealth of previously generated connnectomic, morphological, and functional measurements. However, techniques for this 


but existing techniques for spatial subregion detection are poorly suited for analysis of large-scale, 3D imaging data. For example, one common class of methods utilizes spatial Gaussian processes[@townes_nonnegative_2023] with $O(n^3)$ scaling, which is undesirable for large spatial (3D) imaging datasets. Other classes of promising methods would require assumptions to be made about the size of the spatial neighborhood for cellular interaction, which is difficult to assess with serial sections[@brbic_annotation_2022].



Single-cell sequencing studies[@tasic_shared_2018; @zeisel_molecular_2018; @bandler_single-cell_2022] in the mouse have made significant progress towards charting cell type gradients as defined by transcriptomic content; likewise, computational efforts have been developed for very fine-grained connectivity mapping and clustering[@knox_high-resolution_2018]. Molecular atlases such as the Allen Institute's Common Coordinate Framework[@wang_allen_2020] (CCF) atlas are critical to help organize inquiries into these phenomena by facilitating comparisons across studies and spatial scales. One strength of these atlases is their extensive manual curation, starting from the level of image registration and quality-checking towards consensus decisions by expert anatomists to delineate subregions and their boundaries.

# Aim 1: establish a method for the parcellation and analysis of transcriptomic imaging data

# Aim 2: establish an analysis pipeline for the integration of transcriptomic and connectomic imaging data 
