---
title: "Parcellation of the mouse brain using molecular imaging"
author: Alex Lee
bibliography: references.bib
csl: nature.csl
format:
  pdf:
    toc: false
    number-sections: true
    link-citations: true
    colorlinks: true
---

# Specific Aims

&nbsp;&nbsp;&nbsp;&nbsp;Parallel technological advances in molecular biology and microscopy offer modern scientists the opportunity to make spatially resolved measurement of large numbers of proteomic or transcriptomic features at unprecedented speed and scale. These high-fidelity molecular mapping techniques are promising to catalog the enormous diversity of cell types in the complex organs like the brain, which has been recognized since the early work of Ramon y Cajal [@y_cajal_texture_1999]. In particular, large scale collaborations such as those facilitated by the NIH BRAIN initiative[@noauthor_nih_2022] are generating large, multimodal and cross-species datasets towards precisely elucidate the structural organization of the brain and its relationship to brain function. 

&nbsp;&nbsp;&nbsp;&nbsp;Two critical areas of interest in the neuroscience community are diversity in gene expression[@tasic_shared_2018] and connectivity patterns[@oh_mesoscale_2014; @sun_integrating_2021], and how they interact in the circuitry of the brain. A variety of unsupervised and supervised machine learning techniques have been developed for extraction of subregions from spatial imaging data[@walker_deciphering_2022], however these methods typically target 2D tissue sections, making hem unsuited for analysis of 3D volumetric data. In addition, there are few techniques suited for multimodal data integration of the type that would be useful for integration of connectivity and spatial information, and most methods focus on spatial transcriptomics exclusively. 

&nbsp;&nbsp;&nbsp;&nbsp;I propose to address this unmet need by developing computational methods for machine learning-based spatial subregion detection, with applicability to existing and in-development datasets from the Allen Institute for Brain Science. I will use these techniques to map the organization of functional subregions in high resolution spatial transcriptomics datasets, and use interpretable statistical techniques to understand the cell type contributions to these subregions. I will then make use of these techniques to integrate this spatial transcriptomics dataset with the existing Allen Mouse Connectivity Atlas[@oh_mesoscale_2014], identifying genetic correlates of differential connectivity in transcriptomically similar cells.

## Aim 1: establish a method for the parcellation and analysis of transcriptomic imaging data

&nbsp;&nbsp;&nbsp;&nbsp;I hypothesize that the mouse brain's subregions vary in terms of only a small number of the different cell types. To investigate this I will build on previous models for interpretable, stability-driven automatic subregion detection[@wu_stability-driven_2016] and extend them to the setting of spatial transcriptomics datasets, which are both high dimensional and very sparse. I will then characterize these subregions by analyzing the relative contribution of different cell types to these different identified regions.

## Aim 2: establish an analysis pipeline for the integration of transcriptomic and connectomic imaging data 

&nbsp;&nbsp;&nbsp;&nbsp;I will study the relationship between gene expression and connectivity by developing statistical methods to test whether particular genes are indicative of differential connectivity. I will benchmark methods on their ability to capture previously identified connectivity-genetic relationships and test whether single cell features or regional features are more predictive of differential connectivity. 

## Significance: 
 
some other stuff

# Background

&nbsp;&nbsp;&nbsp;&nbsp;Since the inception of the neuroanatomy field[@y_cajal_texture_1999], it has been recognized that although cells can be broadly grouped into large categories (like neurons, or glia), even visual inspection of silver-stained cells identifies a diverse array of subtypes that differ along morphologic and connectomic characteristics. Understanding the role of this diversity in facilitating the computational complexity of the brain is one of the fundamental projects of modern neuroscience research. 

&nbsp;&nbsp;&nbsp;&nbsp;Large scale molecular neuroscience projects 


but existing techniques for spatial subregion detection are poorly suited for analysis of large-scale, 3D imaging data. For example, one common class of methods utilizes spatial Gaussian processes, [@townes_nonnegative_2023], which features $\mathbf{O}(N^3)$ scaling, which is undesirable for large spatial (3D) imaging datasets. Other classes of promising methods would require assumptions to be made about the size of the spatial neighborhood for cellular interaction, which is difficult to assess with serial sections[@brbic_annotation_2022].


The ubiquity of high-fidelity molecular imaging data provides a unique opportunity to study the correspondence between these expert annotations and structures and patterns that can be identified in these data using modern machine learning techniques. 

Single-cell sequencing studies[@tasic_shared_2018; @zeisel_molecular_2018; @bandler_single-cell_2022] in the mouse have made significant progress towards charting cell type gradients as defined by transcriptomic content; likewise, computational efforts have been developed for very fine-grained connectivity mapping and clustering[@knox_high-resolution_2018]. Molecular atlases such as the Allen Institute's Common Coordinate Framework[@wang_allen_2020] (CCF) atlas are critical to help organize inquiries into these phenomena by facilitating comparisons across studies and spatial scales. One strength of these atlases is their extensive manual curation, starting from the level of image registration and quality-checking towards consensus decisions by expert anatomists to delineate subregions and their boundaries.

# Aim 1: establish a method for the parcellation and analysis of transcriptomic imaging data

# Aim 2: establish an analysis pipeline for the integration of transcriptomic and connectomic imaging data 
